docs/rules/no-dynamic-require.md

```diff
@@ -1,4 +1,6 @@
-# import/no-dynamic-require: Forbid `require()` calls with expressions
```

Comment:
preserve

Reply:
There's an equivalent sentence a few lines down.

Reply:
Per https://github.com/import-js/eslint-plugin-import/pull/2582#discussion_r1008920390, i think the exact description phrase text needs to appear in the doc somewhere.